frontend/src/components/Form/CustomSelectionButton/CustomSelectionButton.tsx

```diff
@@ -0,0 +1,57 @@
+import { MaterialSymbol } from 'material-symbols'
+import { Color } from '../../../types/color'
+import { Icon } from '../../Icon/Icon'
+
+export type CustomSelectionButtonProps = {
+	/** If the button is active or not. */
+	active: boolean
+	/** The color of the border and icon when active */
+	activeColor?: Color
+	/** The icon when active. */
+	activeIcon?: MaterialSymbol
+	/** The function that is called when the button is clicked. */
+	onClick: (event: React.MouseEvent<HTMLButtonElement>) => void
+	/** The headline in the button. */
+	headline: string
+	/** The description in the button. */
+	description: string
+	/** For testing. */
+	datacy?: string
+}
+
+/**
+ * A custom selection button that can be used in a form for a selection between states.
+ */
+export const CustomSelectionButton: React.FC<CustomSelectionButtonProps> = ({
+	active,
+	activeColor = 'yellow',
+	activeIcon = 'check_circle',
+	onClick,
+	headline,
+	datacy,
+	description,
+}) => {
+	return (
+		<button
+			datacy={datacy}
+			type="button"
+			onClick={onClick}
+			disabled={active}
+			className={`flex flex-col items-center border-2 transition duration-200 ease-in-out ${
+				active ? `border-${activeColor}-500` : 'border-gray-100'
+			} rounded-lg px-3 pb-5 pt-2 md:px-4 md:pb-6 md:pt-3`}
```

Comment:
Der Delay nur auf der Border wirkt bissle komisch. Funktioniert der Fade In vllt auch für das Icon?